airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/python_connectors.py

```diff
@@ -292,5 +292,11 @@ def get_test_steps(context: ConnectorTestContext) -> STEP_TREE:
                 args=lambda results: {"connector_under_test_container": results[CONNECTOR_TEST_STEP_ID.BUILD].output[LOCAL_BUILD_PLATFORM]},
                 depends_on=[CONNECTOR_TEST_STEP_ID.BUILD],
             ),
+            StepToRun(
```

Comment:
❓ Do we have any reason to not run these tests we need to plan for?

Reply:
By default, [they'll always be skipped unless specifically selected](https://github.com/airbytehq/airbyte/pull/38816/files#diff-3a2feb8da6d3b4507f7a42303e54de4ac916308de4501f2e77d6ee46c97b45edR118). Whenever we're ready for broader automation we'll want to talk about the frequency.